### DISCUSSION TITLE
V5 is coming!

### DISCUSSION THREAD

**Discussion by cyp3rius** [VirtusLab-Open-Source/strapi-plugin-comments#253] [Ideas]
Hello Folks!

As Strapi V5 beta was published we're already started the v5 re-implementation of the plugin :) 

It's time to ask you, the community, what features you're missing the most and would like to see in upcoming releases of the plugin? :)

We're going to build a community driven backlog combined with our own we're working on right now. Time start!

**Comment by tobiaswu** [comment]
Great. I'd like to have a way to send email notifications to a user when someone replies to their comment and starts a thread. When the thread continues send notifications to all users who participated. I initially thought I could connect it with the Strapi email plugin.

**Comment by vitakili** [comment]
Hey,
Thanks a lot for you work on the comments plugin.
I know that for Strapi V5 its still in the beta, but still I would like to ask if comment removal from the client is working? As Im unable to correctly send the DELETE request

`export async function RemoveComment(postId: number, commentId: number, authorId: string | number) {
  const relation = `api::article.article:${postId}`;

  const res = await fetch(`${STRAPI_URL}/api/comments/${relation}/comment/${Number(commentId)}?authorId=${authorId}`, {
    headers: {
      Accept: "application/json",
      "Content-Type": "application/json",
    },
    method: "DELETE",
  });

  if (!res.ok) {
    throw new Error("Failed to remove comment");
  }

  const data = await res.json();
  return data;
}`

*EDIT:*
solution was to change getRemoveCommentSchema commentId to z.union([z.string(), z.number()])

**Reply by cyp3rius** [reply]
It's under the development yet, we're moving from `id` based queries to `documentId` at this very moment

**Comment by DezyNation** [comment]
What's the status?

**Comment by DezyNation** [comment]
Great!
Please ping me once done.

**Comment by cyp3rius** [comment]
Subscribe for watching the repo and you'll be notified about the releases :)
